Kickstarter-iOS/Views/Controllers/CommentRepliesViewController.swift

```diff
@@ -1,4 +1,68 @@
-import Foundation
+import KsApi
+import Library
+import Prelude
+import ReactiveSwift
 import UIKit
 
-final class CommentRepliesViewController: UITableViewController {}
+final class CommentRepliesViewController: UITableViewController {
+  // MARK: Properties
+
+  private let dataSource = CommentRepliesDataSource()
+  private let viewModel: CommentRepliesViewModelType = CommentRepliesViewModel()
+
+  // MARK: - Accessors
+
+  internal static func configuredWith(comment: Comment) -> CommentRepliesViewController {
+    let vc = CommentRepliesViewController.instantiate()
+    vc.viewModel.inputs.configureWith(comment: comment)
+
+    return vc
+  }
+
+  // MARK: - Styles
+
+  internal override func bindStyles() {
+    super.bindStyles()
+
+    _ = self.tableView |> tableViewStyle
+  }
+
+  // MARK: Lifecycle
+
+  internal override func viewDidLoad() {
+    super.viewDidLoad()
+
+    // TODO: Internationalize
+    self.navigationItem.title = "Replies"
```

Comment:
I think the convention we use for strings that have not been localized yet is to leave the TODO as you did but also set the string with the `localizedString` method.